assets/stylesheets/application.scss

```diff
@@ -1,3 +1,7 @@
+@import "tailwindcss/base";
+@import "tailwindcss/components";
```

Comment:
Do you have any problems when running on development mode and try changing some code?
I had a problem when running with `--watch`, and it took 16 seconds to rebuild the assets

Reply:
@hoangmirs I do have those problems 🤣